components/validator/challenger.go

```diff
@@ -109,6 +110,13 @@ func NewChallenger(ctx context.Context, cfg Config, l log.Logger, m metrics.Metr
 		return nil, fmt.Errorf("failed to get l2 block time: %w", err)
 	}
 
+	cCtx, cCancel = context.WithTimeout(ctx, cfg.NetworkTimeout)
+	defer cCancel()
```

Comment:
Could you change `cCancel()` to `defer cCancel()` in L94, L101, L108? I think this change is missed in [this PR](https://github.com/kroma-network/kroma/pull/139).